lib/msal-browser/src/network/FetchClient.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+import { INetworkModule } from "msal-common";
+
+export class FetchClient implements INetworkModule {
+
+    /**
+     * XHR client for JSON endpoints
+     * https://www.npmjs.com/package/async-promise
+     * @param url 
+     * @param requestParams 
+     * @param enableCaching 
+     */
+    async sendRequestAsync(url: string, requestParams: RequestInit, enableCaching?: boolean): Promise<any> {
+        return await fetch(url, requestParams).then(resp => {
```

Comment:
`fetch` is not supported in IE11. This means we need to either:

1. Provide a fallback for when its not available
2. Use a library like axios or request instead.
3. Provide a polyfill e.g. https://github.com/github/fetch
4. Ask apps to provide a polyfill if they wanted it supported (like we do for Promises).

I would personally prefer we provide the polyfill (we should do this for Promises in V2 as well).

Reply:
This is a bit unrelated to the fetch comments, but additionally here (Jason correct me if I'm wrong) I believe you never need an `await` with a return statement, as it's effectively baked in to `return` for an `async` function. I also feel like having a `.then` after this return/await kind of defeats the point of using async/await to begin with.

Reply:
Yeah, generally speaking its functionally equivalent to not providing it.

And yeah, might as well lean into async/await:

```js
const response = await fetch(url, requestParams);
return resonse.json();
```